tests/baselines/reference/deeplyDependentLargeArrayMutation2.types

```diff
@@ -0,0 +1,3606 @@
+>cvPos : number
+>cvPos + 18 : number
+>cvPos : number
+>18 : 18
+
+            //S
+            closetVertices[cvPos] = closetVertices[cvPos + 9] = oX;
+>closetVertices[cvPos] = closetVertices[cvPos + 9] = oX : any
+>closetVertices[cvPos] : any
+>closetVertices : any[]
+>cvPos : number
+>closetVertices[cvPos + 9] = oX : any
+>closetVertices[cvPos + 9] : any
+>closetVertices : any[]
+>cvPos + 9 : number
+>cvPos : number
+>9 : 9
+>oX : any
+
+            closetVertices[cvPos + 1] = closetVertices[cvPos + 10] = mY;
+>closetVertices[cvPos + 1] = closetVertices[cvPos + 10] = mY : number
+>closetVertices[cvPos + 1] : any
+>closetVertices : any[]
+>cvPos + 1 : number
+>cvPos : number
+>1 : 1
+>closetVertices[cvPos + 10] = mY : number
+>closetVertices[cvPos + 10] : any
+>closetVertices : any[]
+>cvPos + 10 : number
+>cvPos : number
+>10 : 10
+>mY : number
+
+            closetVertices[cvPos + 2] = closetVertices[cvPos + 11] = qZ;
+>closetVertices[cvPos + 2] = closetVertices[cvPos + 11] = qZ : any
+>closetVertices[cvPos + 2] : any
+>closetVertices : any[]
+>cvPos + 2 : number
+>cvPos : number
+>2 : 2
+>closetVertices[cvPos + 11] = qZ : any
+>closetVertices[cvPos + 11] : any
+>closetVertices : any[]
+>cvPos + 11 : number
+>cvPos : number
+>11 : 11
+>qZ : any
+
+            //T
+            closetVertices[cvPos + 3] = oX;
+>closetVertices[cvPos + 3] = oX : any
+>closetVertices[cvPos + 3] : any
+>closetVertices : any[]
+>cvPos + 3 : number
+>cvPos : number
+>3 : 3
+>oX : any
+
+            closetVertices[cvPos + 4] = nY;
+>closetVertices[cvPos + 4] = nY : number
+>closetVertices[cvPos + 4] : any
+>closetVertices : any[]
+>cvPos + 4 : number
+>cvPos : number
+>4 : 4
+>nY : number
+
+            closetVertices[cvPos + 5] = qZ;
+>closetVertices[cvPos + 5] = qZ : any
+>closetVertices[cvPos + 5] : any
+>closetVertices : any[]
+>cvPos + 5 : number
+>cvPos : number
+>5 : 5
+>qZ : any
+
+            //P
+            closetVertices[cvPos + 6] = closetVertices[cvPos + 12] = oX;
+>closetVertices[cvPos + 6] = closetVertices[cvPos + 12] = oX : any
+>closetVertices[cvPos + 6] : any
+>closetVertices : any[]
+>cvPos + 6 : number
+>cvPos : number
+>6 : 6
+>closetVertices[cvPos + 12] = oX : any
+>closetVertices[cvPos + 12] : any
+>closetVertices : any[]
+>cvPos + 12 : number
+>cvPos : number
+>12 : 12
+>oX : any
+
+            closetVertices[cvPos + 7] = closetVertices[cvPos + 13] = nY;
+>closetVertices[cvPos + 7] = closetVertices[cvPos + 13] = nY : number
+>closetVertices[cvPos + 7] : any
+>closetVertices : any[]
+>cvPos + 7 : number
+>cvPos : number
+>7 : 7
+>closetVertices[cvPos + 13] = nY : number
+>closetVertices[cvPos + 13] : any
+>closetVertices : any[]
+>cvPos + 13 : number
+>cvPos : number
+>13 : 13
+>nY : number
+
+            closetVertices[cvPos + 8] = closetVertices[cvPos + 14] = iZ;
+>closetVertices[cvPos + 8] = closetVertices[cvPos + 14] = iZ : any
+>closetVertices[cvPos + 8] : any
+>closetVertices : any[]
+>cvPos + 8 : number
+>cvPos : nu
```

Comment:
Do we not have a reduced repro we can baseline? This seems more appropriate for the RWC tests. I can't imagine anyone actually reviewing this to any depth.

Reply:
I simplified it, but it's not much shorter, just way less obvious what it's motivation is. To hopefully prevent someone brushing off a regression in it in the future, I've included a comment to the effect of why the test exists and what more obvious pattern it represents.